src/js/PhotoSphereViewer.core.js

```diff
@@ -74,8 +74,8 @@ PhotoSphereViewer.prototype._loadXMP = function() {
     throw new PSVError('Cannot load image');
   };
 
-  if (true === this.prop.cacheTextures && 'undefined' != typeof this.prop.loadedXMP[this.config.panorama]) {
-    defer.resolve(this.prop.loadedXMP[this.config.panorama]);
+  if (true === this.config.cacheTextures && 'undefined' != typeof this.config.loadedXMP[this.config.panorama]) {
+    defer.resolve(this.config.loadedXMP[this.config.panorama]);
```

Comment:
Yeah im a bit in a rush and did a quick copy'n'paste.
Now I need to work on another project, i'll do a proper update in the next days